src/app/pages/tutorial/tutorial.ts

```diff
@@ -27,10 +28,13 @@ export class TutorialPage {
       .then(() => this.storage.set('ion_did_tutorial', true));
   }
 
-  onSlideChangeStart(event) {
-    event.target.isEnd().then(isEnd => {
-      this.showSkip = !isEnd;
-    });
+  setSwiperInstance(swiper: Swiper) {
+    this.slides = swiper;
+  }
+
+  onSlideChangeStart() {
+    this.showSkip = !this.slides.isEnd;
+    this.cd.detectChanges();
```

Comment:
Why do we manually call change detection here?

Reply:
When `showSkip` is updated, Angular doesn't automatically update the view and the Skip button stays visible.

Reply:
Looks like Swiper events are emitted outside of NgZone intentionally: https://swiperjs.com/angular#swiper-component-events

Good to leave this as is 👍